wolfcrypt/src/aes.c

```diff
@@ -2109,17 +2189,17 @@ int wc_AesSetIV(Aes* aes, const byte* iv)
 
         HAL_CRYP_Init(&hcryp);
 
-        while (sz > 0) {
+        while (len > 0) {
```

Comment:
Is > 0 the right comparison here?  If someone tries to encrypt 4 bytes we'll encrypt/write 16 bytes to output.